packages/bundle-odoo-website-editor/OdooWebsiteEditor.ts

```diff
@@ -57,14 +57,34 @@ interface OdooWebsiteEditorOptions {
     snippetMenuElement?: HTMLElement;
     snippetManipulators?: HTMLElement;
     template?: string;
-    toolbar?: boolean;
+    layout?: ToolbarLayout;
```

Comment:
You mean `toolbarLayout`, right ? Or whatever it's called in the `Toolbar` plugin configuration. My point is that `layout` in the context of the editor itself is already something else.